src/video-player.utils.js

```diff
@@ -41,7 +41,7 @@ export const getResolveVideoElement = (elem) => {
 
     try {
       elem = document.querySelector(`#${cssEscape(id)}`) || videojs.getPlayer(id);
-    } catch (e) {
+    } catch (err) { // eslint-disable-line no-unused-vars
       elem = null;
```

Comment:
I would console.warn the error instead of the eslint disable line ... since anyway we have other console.warn statements .... later we need to start thinking how we should report these errors somewhere

Reply:
see previous